src/script.js

```diff
@@ -17,6 +17,7 @@ class Script {
     this.body = null;
     this.ext = (os.platform() === 'win32') ? ('cmd') : ('sh');
     this.prefix = (os.platform() === 'win32') ? (['echo off']) : (['#!/bin/bash -e']);
+    this.suffix = (os.platform() === 'win32') ? ([]) : ([]);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This conditional operation returns the same value whether the condition is "true" or "false". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS3923)